app/src/main/java/com/kamron/pogoiv/logic/PokemonNameCorrector.java

```diff
@@ -87,12 +87,12 @@ public PokeDist getPossiblePokemon(String poketext, String candytext, Optional<I
 
         //4.  check correction for Eevee’s Evolution
         if (guess.pokemon == null) {
-            HashMap<String, String> EeveelutionCorrection = new HashMap<String, String>();
-            EeveelutionCorrection.put("Rainer", pokeInfoCalculator.get(133).name); //Vaporeon
-            EeveelutionCorrection.put("Sparky", pokeInfoCalculator.get(134).name); //Jolteon
-            EeveelutionCorrection.put("Pyro", pokeInfoCalculator.get(135).name); //Flareon
-            if (EeveelutionCorrection.containsKey(poketext)) {
-                poketext = EeveelutionCorrection.get(poketext);
+            HashMap<String, String> eeveelutionCorrection = new HashMap<>();
```

Comment:
@NightMadness For your information: by convention, variable names (like methods and fields) start by convention lowercase, so it's easier to distinguish those names from class names.